fdupes.c

```diff
@@ -140,6 +145,38 @@ char *fmttime(time_t t) {
   return buf;
 }
 
+// current timestamp in milliseconds
+uint64_t now64(void)
```

Comment:
This was copied from another project of mine where I need a monotonic clock. I assume this project does not compile on Windows, but I did leave the WIN32 part